hawc/apps/common/serializers.py

```diff
@@ -0,0 +1,9 @@
+from rest_framework import serializers
+
+
+class UnusedSerializer(serializers.Serializer):
+    """
+    Nondescript serializer. Used in instances where a view does not use a serializer but one needs to be set, such as for OpenAPI generation.
```

Comment:
one more comment, then good to merge - can you please wrap the text here, instead of relying on your editor to wrap?